### DISCUSSION TITLE
How to customize customFields input widgets

### DISCUSSION THREAD

**Discussion by skid** [vendurehq/vendure#1055] [Q&A] [answered]
Is there a way to control what widget is displayed in the Admin UI for custom fields? For example, the Admin UI already contains a rich text editor for the product description, so is there a configuration-only way to show a RTE for a `text` custom field?

**Comment by michaelbromley** [comment, marked as answer]
Not a config-only option right now. You _can_ do it but you need to extend the Admin UI. Open issue for this here: https://github.com/vendure-ecommerce/vendure/issues/722
